### ISSUE TITLE
Edit language about response time interval for permits submitted at the ABC Portal (SR) (LP) (On PROD)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#9372]
### Update Portal text (displayed after submittal) about response times for ROW permit applications. 

**Acceptance Criteria**

- The help text on the 'Permit in Queue' box on the Application submitted page on the portal should be updated as mentioned below. 

- **Current Text:** "The application has been submitted for review by permit specialist, customer should receive a response within 1 to 5 business days."

- **Updated Text:** "The application has been submitted for review by a permit analyst.  Customer should receive a response within 5 business days.  Business days exclude weekends and holidays."

- This applies for DS, EX, RW and PSM folders (across all of Right of way department's folders). 

Screenshot provided below .![Portal help text Austin.JPG](https://images.zenhubusercontent.com/5f50fe1f66d1a40f7f557fb4/47848ecb-694e-40c6-af95-2502c9eeb821) 

_Darren asked for this update to apply to UC submittals also._

----
### Form Submittal: 

> What application are you using?

AMANDA

> Describe the problem.

We recently noticed that we an applicant submits their permit request via ABC Portal, one of the notes that displays states: "The application has been submitted for review by permit specialist, customer should receive a response within 1 to 5 business days."

While there are some permits which are reviewed within 1 business day (i.e. emergency and secondary requests), the vast majority are reviewed closer to the 5 business day mark.  We would like to edit this language to more accurately reflect the review timeline.

> Describe the outcome you'd like to see when this feature is implemented.

We would like to see the language changed to: "The application has been submitted for review by a permit analyst.  Customer should receive a response within 5 business days.  Business days exclude weekends and holidays."

> Describe any workarounds you currently have in place or alternative solutions you've considered.

We have no workarounds.  The current text could explain why we have so many ppl reach out to us just a couple days after submittal with complaints about delays.

> Is there anything else we should know?

N/A

> Requested By
Paloma A.

Request ID: DTS22-104205

**Comment by TracyLinder** [comment]
**Scoping Questions:**

1. Get a screenshot of the page Paloma is referring to where the text is. 
2. Do we know if this is an issue for all of ROW's portal applications? _(It's possible this is an enterprise default, which complicates things.)_

**Comment by TracyLinder** [comment]
@Nadin-Nader I added questions I'd like asked during scoping. I'm putting this in needs scoping and it can be scoped per normal prioritization. This will not be developed during the V7 upgrade.

**Comment by TracyLinder** [comment]
I asked Paloma for a screenshot.

**Comment by vschandramouli** [comment]
Updated AC, moving to Backlog.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Analysis is in Progress

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Manual DEV DP submitted. Kindly approve and create task id for your reference.
Note: It’s a manual commit on Smartlet page

[ATD 9372 - LIT DEV Deployment Plan - Manual - Smartlets .docx](https://github.com/cityofaustin/atd-data-tech/files/10854515/ATD.9372.-.LIT.DEV.Deployment.Plan.-.Manual.-.Smartlets.docx)

**Comment by Nadin-Nader** [comment]
This has been sent to CTM to deploy the manual smartlet changes to DEV under **SCTASK0291253**

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Tested in DEV and working as expected. Here are the sample screen shots for EX/RW/DS permits. PSM folder not able to test due to permissions. However This change will be applicable to all folders as it is an enterprise level change. 

![image](https://user-images.githubusercontent.com/44907715/223494940-84467009-0166-4a2b-9faa-7d272a5bb423.png)

![image](https://user-images.githubusercontent.com/44907715/223494990-7ae019b6-3cd2-41cb-aeba-fe7fdabce17a.png)

![image](https://user-images.githubusercontent.com/44907715/223495032-3caaa55b-f9bf-40ce-bdac-116ed3089cec.png)

**Comment by TracyLinder** [comment]
@Sreelatha-BlazeKannan I don't understand why you'd need permission to apply for a PSM on portal DEV. Is there something we need to update?

**Comment by TracyLinder** [comment]
@Sreelatha-BlazeKannan the screenshots look good. 

### Please promote to TEST.

**Comment by TracyLinder** [comment]
@Sreelatha-BlazeKannan I decided not to promote to TEST until ROW approves. I will post in the TEAMS channel. Sorry for the confusion.

**Comment by TracyLinder** [comment]
@Sreelatha-BlazeKannan please update the text to read "The application has been submitted for review by a permit analyst. Customer should receive a response within 5 business days. Business days exclude weekends and holidays."

Paloma also asked that we test to see if this change will appear on all portal applications, or just these few listed. 

Thank you!

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - DEV manual DP of #9372 as per updated AC changes. Kindly approve and create task id for me.
This is a Smartlet page manual changes. 
[ATD 9372 - LIT DEV Deployment Plan - Manual - Smartlets v1.1.docx](https://github.com/cityofaustin/atd-data-tech/files/10925132/ATD.9372.-.LIT.DEV.Deployment.Plan.-.Manual.-.Smartlets.v1.1.docx)

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan  i have sent this to CTM for the Smartlet changes under **SCTASK0293624**

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this has been deployed to DEV please UAT

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Tested in DEV and working as expected. Here are the sample screen shots for EX/RW/DS permits. PSM folder not able to test as I do not have insert permissions (screen shot attached). However This change will be applicable to all folders as it is an enterprise level change.

![image](https://user-images.githubusercontent.com/44907715/225120314-bd0b6ec0-693d-4cde-9a9e-85ab8d309e1e.png)


![image](https://user-images.githubusercontent.com/44907715/225120347-e79b5788-c24b-4e22-83e1-80afeccea36c.png)


![image](https://user-images.githubusercontent.com/44907715/225120385-f3c4163c-58fd-46ef-a017-4fe35622f174.png)


![image](https://user-images.githubusercontent.com/44907715/225120413-18145777-000b-41e3-ad5e-8dac85589f17.png)

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Paloma said that they will get back on this.

**Comment by Nadin-Nader** [comment]
Update Text from "The application has been submitted for review by a permit analyst." to "The application has been submitted."

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - DEV manual DP submitted. Kindly approve and create task id for me.

Note : This is Smartlet page manual changes. 

From:  The application has been submitted for review by a permit analyst.

To: The application has been submitted.

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this has been sent to CTM to deploy the Smarlet changes to DEV under **SCTASK0300916**

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Verified in Dev, Looks good. Ready for Test Deployment

**Comment by Lpriya-LIT** [comment]
![image.png](https://images.zenhubusercontent.com/611eac1954217f265acc0499/a68d79c7-00d7-46bd-adde-47bb8e8800a6)

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Created Test promotion request

**Comment by Nadin-Nader** [comment]
Assigned to Swapna but has not been deployed to TEST yet

**Comment by Nadin-Nader** [comment]
Sent Kyle a message stating that this was ready for UAT on TEST

**Comment by Nadin-Nader** [comment]
Still not deployed to PROD

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Changes deployed to Prod , Ready for UAT

**Comment by Nadin-Nader** [comment]
Sent Message to Kyle and ROW Team to please UAT and let us know if this is good to close or not now that it is on PROD. Waiting on Response

**Comment by Nadin-Nader** [comment]
This has been approved by ROW, closing
